streams/src/test/java/org/apache/kafka/streams/integration/ResetIntegrationTest.java

```diff
@@ -287,6 +277,31 @@ private Properties prepareTest() throws Exception {
     }
 
     private void prepareInputData() throws Exception {
+        try {
```

Comment:
I like to be conservative -- if deleting one topic fails because the topic is not there, we still might need to delete other topics. (I guess, right now, the overall structure of the test does not allow for this scenario though).